actionview/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Add queries count to template rendering instrumentation
+
+    ```
+    # Before
+    Completed 200 OK in 3804ms (Views: 41.0ms | ActiveRecord: 33.5ms | Allocations: 112788)
+
+    # After
+    Completed 200 OK in 3804ms (Views: 41.0ms | ActiveRecord: 33.5ms | SQL Queries: 2 (1 cached) | Allocations: 112788)
```

Comment:
Would it make sense to group it with `ActiveRecord` like `ActiveRecord 33.5ms (2 queries, 1 cached)`? "SQL Queries" could be related to anything (also outside of ActiveRecord).

Reply:
Yeah, was thinking about this too 👍 Let's see if core people agree this PR is a good idea and will then work on this.